CONTRIBUTING.md

```diff
@@ -1,91 +1,101 @@
+The more correct way to see payment is "compensation". By earning BSQ, you invest in the Bisq project (just like other startups do it). But make no mistake, the DAO has only been operating for a few months now. Eventually, Bisq will create serious revenue.
 
+**Tools?** If you want to develop code, we expect you to know your ways around Git and Java. Then comes testing, JavaFX, Gradle, Github, ... As for non-functional skills, we love to see people knowing their ways around P2P networks, cryptography, security, privacy, ...
 
-## Contributor Workflow
+**Carreer?** Start small, go bigger. If we like your work, we might just ask you to take on a more official role. Maybe someday you are the guy improving this document (again).
 
-All Bisq contributors submit changes via pull requests. The workflow is as follows:
 
- - Fork the repository
- - Create a topic branch from the `master` branch
- - Commit patches
- - Squash redundant or unnecessary commits
- - Submit a pull request from your topic branch back to the `master` branch of the main repository
- - Make changes to the pull request if reviewers request them and __**request a re-review**__
 
-Pull requests should be focused on a single change. Do not mix, for example, refactorings with a bug fix or implementation of a new feature. This practice makes it easier for fellow contributors to review each pull request on its merits and to give a clear ACK/NACK (see below).
 
+Still here? Nice. You are hired. On to the grind.
 
-## Reviewing Pull Requests
+# Getting started
 
-Bisq follows the review workflow established by the Bitcoin Core project. The following is adapted from the [Bitcoin Core contributor documentation](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review):
+Join Gibhub if you haven't already, fork the Bisq repository, configure your local git ([username](https://help.github.com/articles/setting-your-username-in-git/), [commit signing](https://help.github.com/articles/signing-commits-with-gpg/)), clone your Bisq-fork to disk, do a compile run `./gradlew build`.
 
-Anyone may participate in peer review which is expressed by comments in the pull request. Typically reviewers will review the code for obvious errors, as well as test out the patch set and opine on the technical merits of the patch. Project maintainers take into account the peer review when determining if there is consensus to merge a pull request (remember that discussions may have been spread out over GitHub, mailing list and IRC discussions). The following language is used within pull-request comments:
+Now you are ready to roll. Pick something off the [Good First Issue List](https://github.com/bisq-network/bisq/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), pick any other issue, do some testing, spot a bug, report it and fix it, maybe there is something in the UI that needs cosmetic work.
```

Comment:
Although everyone can do whatever they want, maybe we should point the user more to the tasks we'd like to have them fulfilled. E.g. Take a good first issue or fix a bug.

Reply:
The first item is the good first issues, the second one is fix a bug. Please elaborate.